### ISSUE TITLE
Babel runtime isn't uncluded

### ISSUE THREAD

**Issue by finom** [systemjs/builder#342]
I'm getting 
```
regeneratorRuntime is not defined
```

Even if I'm trying to force include runtime:
```js
builder.buildStatic('app/app.js', 'dist/js/app.js', {runtime: true});
```

Build goes well. The problem is in resulting JS file. Error is thrown when I use async functions.

Version of builder is 0.14. Do you need any additional info?

**Comment by guybedford** [comment]
Interesting - right, this is because when just using regenerator, Babel doesn't report that any helpers were needed at all.

This might actually be a Babel bug that usedHelpers doesn't return `regeneratorRuntime` I've posted https://github.com/babel/babel/issues/2495.

**Comment by finom** [comment]
@guybedford got it. Do you know any quick'n'dirt solution? How can I include the runtime into a bundle manually?

**Comment by guybedford** [comment]
@finom you could replace https://github.com/systemjs/builder/blob/master/lib/compile.js#L371 with `true` or just make sure you use some other helper from Babel.

**Comment by finom** [comment]
@guybedford got it, thanks.

**Comment by guybedford** [comment]
Released in 0.14.8.

**Comment by finom** [comment]
Thanks.
